project/LibDependencies.scala

```diff
@@ -15,8 +15,8 @@ object LibDependencies {
   }
 
   def compileDependencies(playVersion: PlayVersion) = playVersion match {
-    case PlayVersion.Play28 => "com.typesafe.play" %% "play" % "2.8.21"
-    case PlayVersion.Play29 => "com.typesafe.play" %% "play" % "2.9.0"
+    //case PlayVersion.Play28 => "com.typesafe.play" %% "play" % "2.8.21"
+    case PlayVersion.Play29 => "com.typesafe.play" %% "play" % "2.9.5"
```

Comment:
Did you need to do this?

Reply:
I will delete this as well. Thanks.

Reply:
Just the version bump I mean, don’t  delete the line :)